internal_docs/releasing.md

```diff
@@ -29,21 +29,26 @@ v8.0.0 // After implementing backwards incompatible changes
 
 3. Checkout a new branch called release-[new version number].
 
-4. Update the version number in [VERSION.txt](/VERSION.txt) and update "version" in [package.json](/package.json#L4).
+4. Update the [CHANGELOG.md](https://github.com/alphagov/govuk-prototype-kit/blob/main/CHANGELOG.md) by:
```

Comment:
I think we can remove the GitHub part of the URL here

```suggestion
4. Update the [CHANGELOG.md](/CHANGELOG.md) by:
```

Reply:
Cheers! Will note for future reference.